hathor/indexes/rocksdb_tx_group_index.py

```diff
@@ -108,9 +108,15 @@ def remove_tx(self, tx: BaseTransaction) -> None:
             self._db.delete((self._cf, self._to_rocksdb_key(key, tx)))
 
     def _get_from_key(self, key: KT) -> Iterable[bytes]:
```

Comment:
Why not just add a `tx` here?

Reply:
This method implements an abstract method of the `TxGroupIndex`.  Adding a `tx` parameter here means that we should add this to the abstract method and all other classes that extend this `TxGroupIndex` class and this is particular of the rocksdb implementation.

Apart from that, the `get_from_key` method doesn't need any `tx` because it doesn't do any kind of sorting, see the docstring:

> """Get all transactions that have a given key."""

That's why I decided to create a util method instead of changing this one.

Reply:
Got it. Thanks for the explanation.